prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.html

```diff
@@ -14,13 +14,29 @@
                       (navigateEnrollee)="onNavigateEnrollee($event)"
                       (sendBulkEmail)="onSendBulkEmail()"></app-enrollee-table>
 
-  <div class="d-flex justify-content-between align-items-center mt-4">
+  <div class="d-flex justify-content-between align-items-center mt-4 mb-4">
     <span>
+      <app-alert *ngIf="!enrollee?.linkedEnrolleeId && enrollee?.possiblePaperEnrolmentMatch"
```

Comment:
We should try and constrain the alert.  Have it be the same width as the Enrolment overview that is contained underneath it so we have separation between it and the TOA Selection.

![image](https://user-images.githubusercontent.com/599498/144151645-1e3cd56e-3aa3-4098-bc01-4f82e71e7c5b.png)

Reply:
Fixed

Reply:
That's actually not how you're supposed to use `col-6` that's a Bootstrap Grid class and you're using it within Flexbox.  The reason it appears to work is due to the grid being built using flexbox, but `col-6` should always be child of `row` and it also applies a gutter.

This will help with reducing the presence of the alert on the page, provide space between elements, and improving readability.  I took out the alert title since it seems a bit large for our needs and replaced it with a block-level strong container with the title, as well as, a small container for the helper text then we don't need to play with widths to have the justify-content-between work for us since this reduces the content size associated space.  I reduced the text since they found the paper enrollee likely through searching so we can simply point out what they might not know about the adjudication section which will be updated to links in the other PR.

```html
<div class="d-flex justify-content-between align-items-center my-4">
  <span>
    <app-alert *ngIf="!enrollee?.linkedEnrolleeId && enrollee?.possiblePaperEnrolmentMatch"
               type="warning">
      <ng-container #alertContent
                    class="alert-content">
        <strong class="d-block">Possible paper enrolment matches found</strong>
        <small>See the possible matches linked under the Adjudication section.</small>
      </ng-container>
    </app-alert>
    ...
  </span>
  ...
</div>
```

Reply:
![image](https://user-images.githubusercontent.com/599498/144304496-1be8d4ad-8662-40e9-9198-d9941d29187d.png)